### PR TITLE
{devel}[GCCcore/5.4.0] PCRE v8.39

### PR DESCRIPTION
**Pull request by vanzod**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
cermis - Linux debian 8.9, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.9
See https://gist.github.com/e5afdabd463cf0900171f697cf51782c for a full test report.

File: easybuild/easyconfigs/p/PCRE/PCRE-8.39-GCCcore-5.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+toolchain = {'name': 'GCCcore', 'version': '5.4.0'}
+toolchainopts = {'pic': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['ccdf7e788769838f8285b3ee672ed573358202305ee361cfec7a4a4fb005bbc7']
+
+builddependencies = [
+    # use same binutils version that was used when building GCCcore toolchain
+    ('binutils', '2.27', '', True),
```
Comment (boegel) [inline easybuild/easyconfigs/p/PCRE/PCRE-8.39-GCCcore-5.4.0.eb]: @vanzod No need to hardcode to `dummy` toolchain here, and it's OK to make this single-line nowadays imho...

```python
builddependencies = [('binutils', '2.27')]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2072.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/acd2318e1e2ef031b2048575ab620bae for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2532.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/7f2b6c413bc2945a363d36e0c02d9ae9 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @vanzod!

**Pull request merged**